project/make/fedora

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+DEST=$1
+
+# source the .bundle file
+source "$(dirname "$BASH_SOURCE")/.bundle"
+
+# Build docker as a fedora package using FPM and REPREPRO (sue me).
+# bundle_binary must be called first.
+bundle_fedora() {
+	DIR=$DEST/build
+
+	# include udev rules, contributed init, completions & man pages
+	bundle_udev
+	bundle_init
+	bundle_completions
+	bundle_manpages
+
+	# Copy the binary
+	# This will fail if the binary bundle hasn't been built
+	mkdir -p $DIR/usr/bin
+	cp $DEST/../binary/docker-$VERSION $DIR/usr/bin/docker
+
+	# Generate postinst/prerm/postrm scripts
+	cat > $DEST/postinst <<'EOF'
+#!/bin/sh
+set -e
+set -u
+
+if [ "$1" = 'configure' ] && [ -z "$2" ]; then
+	if ! getent group docker > /dev/null; then
+		groupadd --system docker
+	fi
+fi
+
+if ! { [ -x /sbin/initctl ] && /sbin/initctl version 2>/dev/null | grep -q upstart; }; then
+	# we only need to do this if upstart isn't in charge
+	update-rc.d docker defaults > /dev/null || true
+fi
+if [ -n "$2" ]; then
+	_dh_action=restart
+else
+	_dh_action=start
+fi
+service docker $_dh_action 2>/dev/null || true
+
+#DEBHELPER#
```

Comment:
`#DEBHELPER#`? :wink:

We're out of my depth on creating RPMs -- I suggest we find someone who actually knows what they're doing, and get their take on these scripts and how they'll need to change, but it'll help to clarify what our goal is with respect to multiple versions / totally generic, and then maybe ask that same someone if it's a reasonable plan too (since we really, _really_ don't want a repeat of what we're dealing with in our `.deb`s WRT Debian vs Ubuntu).